src/app/guides/how-to-get-funding-for-your-open-source-projects/how-to-get-funding-for-your-open-source-projects.mdx

```diff
@@ -0,0 +1,62 @@
+## How To Get Funding For Your Open Source Projects
+
+# Overview
+
+As wonderful as open source is, one of the biggest challenges maintained projects face is finding funding, especially ones that are sustainable.
+Now, don’t fret! Here are some suggestions on how to get funding for your open source projects as well as some legalities to consider in determining how to get these resources.
+
+## Direct Sponsorships
+
+Direct sponsorship is the most straightforward way to get funding for your project.
+When a user or a different company wants to sponsor your project, they send the funds right to your project’s financial account.
+Here is a list of resources that provide direct sponsorships:
+
+- **GitHub Sponsors**: This is a popular option that creators use to fund their projects.
+  It allows you to choose the number of tiers you charge and rewards that you can offer to potential sponsors.
+  It also provides you with the opportunity to support other developers and organizations, which can be a good way to network with others and gain more support.
+  If you’re interested in using this resource, check out the official website for more information: [https://github.com/sponsors](https://github.com/sponsors)
+- **Open Collective**: This is a company that is devoted to supporting and promoting open source projects.
+  Like GitHub sponsors, they allow you to choose the number of tiers you charge and reward you with potential sponsors.
+  In contrast, Open Source Collective provides information on how the money is being used.
```

Comment:
> provides information on how the money is being used

[Request] This is a little unclear to me, not sure I follow - is it trying to express that Open Collective lets you see granular expenses & transactions for projects? I'd suggest rephrasing to be more explicit on that - arguably GitHub Sponsors does "provide information on how the money is being used".

Reply:
Yes you're correct

Reply:
@JoshuaKGoldberg, this is where I got the 'provides information on how the money is used: [On Github Sponsors Why we welcome it, and how Open Collective is different](https://medium.com/open-collective/on-github-sponsors-5450b63caf7d)